iminuit/_libiminuit.pyx

```diff
@@ -611,6 +611,42 @@ cdef class Minuit:
         matrix = self.matrix(correlation=correlation, skip_fixed=skip_fixed)
         return np.array(matrix, dtype=np.float64)
 
+    def np_values(self):
+        """Parameter values in numpy array format."""
+        import numpy as np
+        return np.asarray(self.args, dtype=np.double)
```

Comment:
Doesn't really matter, but `np.array` here instead of `np.asarray` is probably a tiny bit faster and two characters less. You know that you're calling on a tuple and need to convert to an array, whereas `np.asarray` does a lot of stuff to look at types and numpy array subclasses and ...

Reply:
Good point, I wasn't aware that `array` is faster.